src/frontend/components/LTINav/index.tsx

```diff
@@ -78,6 +85,12 @@ export const LTINav = ({ object: baseObject }: LTINavProps) => {
             <FormattedMessage {...messages.linkPreview} />
           </NavItem>
         )}
+
+        {canAccessDashboard && (
+          <NavItem to={PLAYLIST_ROUTE(appData.modelName)}>
+            <FormattedMessage {...messages.linkPlaylist} />
+          </NavItem>
```

Comment:
You could put it next to
```jsx
<NavItem to={DASHBOARD_ROUTE(appData.modelName)}>
  <FormattedMessage {...messages.linkDashboard} />
</NavItem>
```

Reply:
It would move the tab here too.
I think it should stay after dashboard and preview, but I might be wrong.